save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/fileuploader/CosvFileManagerComponent.kt

```diff
@@ -481,13 +455,3 @@ val cosvFileManagerComponent: FC<Props> = FC {
         }
     }
 }
-
-private fun RawCosvFileDto.notSelectableReason() = when {
```

Comment:
please move this logic to different style of cells with files: need to show such files differently on UI

Reply:
Done

Reply:
what's about tooltip?

Reply:
Done